daemon/src/lib.rs

```diff
@@ -417,9 +417,12 @@ where
 
     #[instrument(skip(self), err)]
     pub async fn propose_settlement(&self, order_id: OrderId) -> Result<()> {
+        // TODO: we need to know which contract symbol the order is for
```

Comment:
Ahh, this is pretty annoying!

I don't want us to have to pass it in via the HTTP API or load it from the database. Shall we move the quote-fetching to the `taker_cfd::Actor` already?